pull-submodules.py

```diff
@@ -177,10 +177,10 @@ def main():
                                     branchToPull = "master"
                                     if branchExistsPcs.stdout.decode("utf-8").rstrip("\n") != "0":
                                         branchToPull = frameworkBranchName
-                                        print(" --> Pulling branch : " + branchToPull + "  ", end='')
+                                    print(" --> Pulling branch : " + branchToPull + "  ", end='')
 
                                     p = subprocess.run(
-                                        f"cd {root}/{submodule} && git checkout origin/{branchToPull} && git pull",  #
+                                        f"cd {root}/{submodule} && git fetch && git checkout {branchToPull} && git pull",  #
```

Comment:
This is not what you mean @jgalan?

Reply:
![imagen](https://user-images.githubusercontent.com/77273709/167915250-93820a72-2fa3-468f-9faf-fec1e2dce981.png)

I can pull a branch in framework and automaticaly the same branch in packages/restG4 for example. If an associated branch doesn't exist it pulls from master.

Reply:
Yes, thats it